guten_reader_FE/src/Components/Settings/Settings.js

```diff
@@ -50,7 +50,11 @@ class Settings extends Component {
               this.setState({ fontSize: itemValue})
           }>
           <Picker.Item label="Size 12" value="12" />
+          <Picker.Item label="Size 16" value="16" />
```

Comment:
@fentontaylor, I think when we talked about the different font size we were going to allow, we settled on 10, 20, or 48 (small, medium, and large). Just looking at my notes. We could include these other options. Frontend what do you think?

Reply:
Those are huge gaps in the sizes though. I think it makes sense to allow for more fine-tuning.

Reply:
When building the component I didn't check the notes, just threw in some place holder values.  So having the ones we actually discussed sounds good :)